manim/mobject/vector_field.py

```diff
@@ -385,6 +382,31 @@ def get_colored_background_image(self, sampling_rate: int = 5) -> Image.Image:
         rgbs = np.apply_along_axis(self.pos_to_rgb, 2, points_array)
         return Image.fromarray((rgbs * 255).astype("uint8"))
 
+    def get_vectorized_rgba_gradient_function(
```

Comment:
**(suggestion)**: an explanation for this method + its parameters would be helpful.

Reply:
Added some docstrings on what I know about it, but I ported it from manimgl and besides some small tweaks I made I don't really know the details of it